packages/docprovider/src/yprovider.ts

```diff
@@ -21,25 +20,26 @@ import { getAnonymousUserName, getRandomColor } from './awareness';
  * We specify custom messages that the server can interpret. For reference please look in yjs_ws_server.
  *
  */
-export class WebSocketProviderWithLocks
-  extends WebsocketProvider
+export class WebSocketProvider
+  extends YWebsocketProvider
   implements IDocumentProvider {
   /**
-   * Construct a new WebSocketProviderWithLocks
+   * Construct a new WebSocketProvider
    *
-   * @param options The instantiation options for a WebSocketProviderWithLocks
+   * @param options The instantiation options for a WebSocketProvider
    */
-  constructor(options: WebSocketProviderWithLocks.IOptions) {
+  constructor(options: WebSocketProvider.IOptions) {
     super(
       options.url,
-      options.contentType + ':' + options.path,
+      options.format + ':' + options.contentType + ':' + options.path,
```

Comment:
This is where we would use the file id rather than the path.

Reply:
Probably, yes.